lib/services/local/resource_test.go

```diff
@@ -0,0 +1,183 @@
+/*
+Copyright 2019 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package local
+
+import (
+	"context"
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/gravitational/teleport/lib/backend"
+	"github.com/gravitational/teleport/lib/backend/lite"
+	"github.com/gravitational/teleport/lib/defaults"
+	"github.com/gravitational/teleport/lib/services"
+	"github.com/gravitational/teleport/lib/services/suite"
+	"github.com/gravitational/teleport/lib/utils"
+	"github.com/jonboulle/clockwork"
+	"gopkg.in/check.v1"
+)
+
+func TestResourceHelpers(t *testing.T) { check.TestingT(t) }
+
+type ResourceSuite struct {
+	bk backend.Backend
+}
+
+var _ = fmt.Printf
+var _ = check.Suite(&ResourceSuite{})
+
+func (r *ResourceSuite) SetUpSuite(c *check.C) {
+	utils.InitLoggerForTests(testing.Verbose())
+}
+
+func (r *ResourceSuite) SetUpTest(c *check.C) {
+	var err error
+
+	clock := clockwork.NewFakeClockAt(time.Now())
+
+	r.bk, err = lite.NewWithConfig(context.TODO(), lite.Config{
+		Path:             c.MkDir(),
+		PollStreamPeriod: 200 * time.Millisecond,
+		Clock:            clock,
+	})
+	c.Assert(err, check.IsNil)
+}
+
+func (r *ResourceSuite) TearDownTest(c *check.C) {
+	c.Assert(r.bk.Close(), check.IsNil)
+}
+
+func (r *ResourceSuite) dumpResources(c *check.C) []services.Resource {
+	startKey := []byte{0}
+	endKey := []byte{
```

Comment:
could be helper function in the backend package `KeyRangeEnd()` or something like that to abstract away the magic constant